lib/iris/coord_systems.py

```diff
@@ -791,6 +791,15 @@ def __repr__(self):
                    self.false_easting, self.false_northing,
                    self.secant_latitudes, self.ellipsoid)
 
+    def __eq__(self, other):
+        sattrs = self.__dict__.copy()
+        oattrs = other.__dict__.copy()
+        s_secant_latitudes = sattrs.pop('secant_latitudes', None)
+        o_secant_latitudes = oattrs.pop('secant_latitudes', None)
+        return (self.__class__ == other.__class__ and
+                sattrs == oattrs and
+                (s_secant_latitudes==o_secant_latitudes).all())
```

Comment:
This is necessary because you're setting the `secant_latitudes` attribute to a numpy array? Have you considered using a property to ensure this attribute is always just a simple tuple?